### PR TITLE
add one more blog

### PR DESCRIPTION
**Pull request by yyfjmb**
merge to master branch

### CODE REVIEW

**Comment by hupili** [comment]
something is wrong

**Comment by andyshuxin** [comment]
Is this a duplicate item?

**Comment by frankinit** [comment]
something is right

File: source/_posts/20151005-read-journalism copy.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+title: "Jackathon #5: Read Journalism"
```
Comment (hupili) [inline source/_posts/20151005-read-journalism copy.md]: look

Comment (andyshuxin) [inline source/_posts/20151005-read-journalism copy.md]: What's this?

Comment (hupili) [inline source/_posts/20151005-read-journalism copy.md]: * 1
* 2
* item 3

Comment (yyfjmb) [inline source/_posts/20151005-read-journalism copy.md]: ## this is a heading

Comment (hupili) [inline source/_posts/20151005-read-journalism copy.md]: @andyshuxin  watchout, something is modifying our repo

Comment (chaotianyi) [inline source/_posts/20151005-read-journalism copy.md]: @frankinit hahah

Comment (Kateliuxue) [inline source/_posts/20151005-read-journalism copy.md]: xixi

Comment (frankinit) [inline source/_posts/20151005-read-journalism copy.md]: can you tell me the way to the number seven middle school?

File: source/_posts/20151005-read-journalism copy.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+The book mainly talks about application of basic Mathematics and Statistics in Journalism.
+Availability
+
+### 中国数据新闻工作坊培训手册
+(Chinese book)
+
+A compilation of a training workshop which taught the whole flow of Data Journalism.
+Availability
+
+Freely available at <http://djchina.org/2015/09/01/china-data-journalism-training-content/>
```
Comment (frankinit) [inline source/_posts/20151005-read-journalism copy.md]: Is this eatable?

**Pull request closed without merging**